modules/hooks.nix

```diff
@@ -36,6 +36,13 @@ in
             "Ansible linter";
           entry = "${tools.ansible-lint}/bin/ansible-lint";
         };
+      black =
+        {
+          name = "black";
+          description = "The uncompromising Python code formatter";
+          entry = "${pkgs.python3Packages.black}/bin/black";
+          files = "\\.py$";
```

Comment:
It's better to use the `python` [`type`](https://pre-commit.com/#filtering-files-with-types) which will also find scripts with python shebangs. (and potentially more?)

Reply:
I didn't know of `types`. Thanks, fixed.